include/libtorrent/file_storage.hpp

```diff
@@ -635,6 +635,11 @@ namespace aux {
 
 		aux::path_index_t get_or_add_path(string_view path);
 
+		bool check_valid_symlink(const aux::file_entry& fe,
+								 const std::unordered_set<std::string>& dir_map,
```

Comment:
this should be one additional level of indentation, not 6

Reply:
sure.
my idea works works strangely)
done.